src/admin-views/venue-meta-box.php

```diff
@@ -163,7 +163,7 @@
 		</tr>
 	<?php
 	}
-	$google_map_link_toggle = ( get_post_status( $event->ID ) != 'auto-draft' || get_post_meta( $event->ID, '_VenueShowMapLink', true ) !== 'false' ) ? true : false;
+	$google_map_link_toggle = ( get_post_meta( $event->ID, '_VenueShowMapLink', true ) !== 'false' || get_post_status( $event->ID ) == 'auto-draft' ) ? true : false;
```

Comment:
We probably should and could for this and a few other things in the same template, don't know if we should _right now_ as part of this lift, though.